### PR TITLE
deps: Run colorama installation on all nodes

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack 6121104

File: circle.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ test:
         parallel: true
     - python setup.py install:
         parallel: true
-    - pip install coala-bears --pre -U:
+    - pip install coala-bears colorama==0.3.6 --pre -U:
```
Comment (Adrianzatreanu) [inline circle.yml]: does it actually work? like arent ``coala-bears`` and ``colorama`` 2 different packages?

Comment (sils1297) [inline circle.yml]: that's simple to check from your local terminal :)

Comment (Adrianzatreanu) [inline circle.yml]: doing, if it works properly im acking this.

**Comment by Adrianzatreanu** [comment]
ack 5651ffd

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**